src/MDMScheduler.m

```diff
@@ -81,16 +81,33 @@ - (MDMSchedulerActivityState)activityState {
 }
 
 - (void)addPlan:(NSObject<MDMPlan> *)plan to:(id)target {
+  [self commonAddPlan:plan toTarget:target log:nil];
+}
+
+- (void)addPlan:(NSObject<MDMPlan> *)plan named:(NSString *)name to:(id)target {
+  NSParameterAssert(name.length > 0);
+  [self commonAddPlan:plan toTarget:target log:[[MDMTransactionLog alloc] initWithPlans:@[plan] target:target name:name removal:FALSE]];
```

Comment:
Created a separate initializer in https://github.com/material-motion/material-motion-runtime-objc/pull/88/commits/15282b9b3d66bc8d71c9bbe4893cadbbf747726b & https://github.com/material-motion/material-motion-runtime-objc/pull/88/commits/b2639f4a41293429680639ccc50c9180b7743aeb